tests/scripts/remote_monitoring_tests/fault_tolerant_tests/SREDebug.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
```

Comment:
1. This needs to be part of $ROOT/scripts directory
2. Please rename this as kruize_db_metrics.sh